packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1037,8 +1038,8 @@ describe("Runtime", () => {
 				});
 			});
 			it("process remote op with unrecognized type and 'Ignore' compat behavior", async () => {
-				const futureRuntimeMessage: ContainerRuntimeMessage = {
-					type: "FROM_THE_FUTURE" as ContainerMessageType,
+				const futureRuntimeMessage: UnknownContainerRuntimeMessage = {
+					type: "__unknown__",
```

Comment:
I liked Jason's point that `__unknown__` is _never_ going to be the value and could be misleading to do so in tests.  Better to do `"FROM_THE_FUTURE" as unknown as "__unknown__"`.

Reply:
Presumably, these tests also want to use a value that is never going to be legitimate (because, if such a value were introduced, the test will no longer be testing "the future" and will be merely testing the present). So, I don't actually see any difference between literally using `__unknown__` and using something that casts to `__unknown__`, except that the latter is longer.

Reply:
I would explicitly use something different. I don't want to see "__unknown__" anywhere else in the codebase. I would even add a linter rule for it. Perhaps "__unknown__" should be even more verbose "__unknown_ContainerRuntimeMessage_type__" or such. Can make is as stupid long as we want because it will never be an actual value.

Reply:
Up to you, my theoretical concern is that someone would look at this and thing the _value_ `__unknown__` means something, but hopefully the underscores would tip them off or something. We're using the _type_ `__unknown__` in place of `Exclude<string, "x" | "y" | "z">` since that's not supported, so any value is a good example.

Reply:
Alright, you've convinced me.